c++/src/kj/common-test.c++

```diff
@@ -216,6 +221,22 @@ TEST(Common, Maybe) {
       ADD_FAILURE();
     }
   }
+
+  {
+    // Test usage of immovable types.
+    Maybe<Immovable> m;
+    KJ_IF_MAYBE(v, m) {
```

Comment:
The CI is failing due to `v` being unused and warnings being treated as errors...